### ISSUE TITLE
OutGate is missing

### ISSUE THREAD

**Issue by develooper1994** [santi-pdp/segan_pytorch#25]
generator.py
class Generator1D(Model):
...
        if post_proc:
            self.comb_net = PostProcessingCombNet(1, 512)
        if out_gate:
            self.out_gate = OutGate(1, 1)  <- There isn't any OutGate
        if big_out_filter:
            self.out_filter = nn.Conv1d(1, 1, 513, padding=513 // 2)
...
